logo/codetally-white.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="7" height="14">
+  <g fill='#fff'>
+    <rect x="0" y="3" width="1" height="7"/>
+    <rect x="2" y="3" width="1" height="7"/>
+    <rect x="4" y="3" width="1" height="7"/>
+    <rect x="6" y="3" width="1" height="7"/>
+    <line x1="0.25" y1="9.75" x2="6.75" y2="3.25" stroke-width="1" stroke="#fff" />
+  </g>
+</svg>
```

Comment:
There isn't consensus about how to handle showing vendor logos, particularly by default. The design goal of shields was to display clean, consistent badges. The social badges have logos, though there are only a couple other badges with logos.

Would you mind removing it until a consensus is reached? It would expedite getting this merged.

Reply:
Thank you Paul. I will update to remove the logo, space and provide testing comments this and re-submit.